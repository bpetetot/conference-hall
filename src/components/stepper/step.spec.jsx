/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import Step from './step'

describe('components/step', () => {
  it('should render', () => {
    const wrapper = shallow(<Step icon="fa-check" step={0} current={0} />)
    expect(wrapper).toMatchSnapshot()
  })
})
