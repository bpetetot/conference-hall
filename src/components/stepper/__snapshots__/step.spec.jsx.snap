// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/step should render 1`] = `
<div
  className="step-icon step-active"
>
  <i
    className="fa-check"
  />
</div>
`;
